### ISSUE TITLE
🚀 Feature Request: LazyLoad background-overlay

### ISSUE THREAD

**Issue by glorpyz** [elementor/elementor#18467]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Speed-test-pages say my <div class="elementor-background-overlay"> is 
"This is the largest contentful element painted within the viewport"

which contains contains a 19 KB background image on repeat.

### Describe the solution you'd like

It would be nice if we could activate LazyLoad on elementor-background-overlay

### Describe alternatives you've considered

E-Addons offers checkboxes on backgroundimages to activate LazyLoad on many places, but it would be nice if Elementor had it.

**Comment by rami-elementor** [comment]
Hi @glorpyz ,

The html attribute `loading="lazy"` can be applied on `<img>` and `<iframe>` tags. Background images are simple `<div>` tags. HTML doesn't allow lazy loading on divs.

Alternative solutions use JavaScript for lazy load. The tradeoff with those solution is that you reduce LCP but you add render blocking resources.

**Comment by jivaneo** [comment]
Hopefully Elementor will provide a real solution in the near future.
Is this even planned to be added in the near future?

**Comment by rami-elementor** [comment]
Yes, we are working on a JS solution for lazy loading background images.

We don't have an official roadmap to release this specific feature, but we have a dedicated team working on ways to improve site performance.

**Comment by jivaneo** [comment]
> Yes, we are working on a JS solution for lazy loading background images.
> 
> We don't have an official roadmap to release this specific feature, but we have a dedicated team working on ways to improve site performance.

Hi there, We'll be waiting for that release then, thanks

**Comment by BrockwayProd** [comment]
Hi, when will you switch to inline in order for any plugin to lazyload images ?

Not your problem anymore once you done that.

Cheers
